public/app/features/explore/Explore.tsx

```diff
@@ -276,6 +285,18 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
     return <NoData />;
   }
 
+  renderCompactUrlWarning() {
+    reportInteraction('grafana_explore_compact_notice');
```

Comment:
@ifrost here is where it is tracked, is there some other sort of tracking we should do?

Reply:
@ifrost Alrighty, I moved it to the initialization, so it will only log out once per mount instead of once per render.